packages/babel-plugin-minify-simplify/README.md

```diff
@@ -2,9 +2,10 @@
 
 This plugin will transform code in mainly two ways:
 
-1. Reduce as much statements as possible into expressions
+1\. Reduce as much statements as possible into expressions
```

Comment:
It's fine, can revert that until we get a more robust fix in place. Just note that this isn't isolated to _just_ the website the numbering is off on npm and GitHub) due to the code block.